js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -606,7 +606,7 @@ keyPassword=Key password
 attributeFriendlyName=Attribute [Friendly Name]
 clearAllFilters=Clear all filters
 scopePermissions.clients.map-roles-composite-description=Policies that decide if an administrator can apply roles defined by this client as a composite to another role
-roleObjectClassesHelp=Object class (or classes) of the role object. It's divided by commas if more classes are needed. In typical LDAP deployment it could be 'groupOfNames'. In Active Directory it's usually 'group'.
+roleObjectClassesHelp=Object class (or classes) of the role object. It's divided by commas if more classes are needed. In typical LDAP deployment it could be 'groupOfNames'. In Active Directory it is usually 'group'.
```

Comment:
```suggestion
roleObjectClassesHelp=Object class (or classes) of the role object. It is divided by commas if more classes are needed. In a typical LDAP deployment, it could be 'groupOfNames'. In Active Directory, it is usually 'group'.
```